sandbox/handlers/data.js

```diff
@@ -0,0 +1,1147 @@
+
+
+const get_message_response_data = [
```

Comment:
Isn't this duplicated content? Can be rm'd?

Reply:
yep, forgot, deleted now